### DISCUSSION TITLE
Customize apps shown in launcher?

### DISCUSSION THREAD

**Discussion by bliedblad** [material-shell/material-shell#746] [Q&A]
With other layouts, I can customize my favorite/most used apps in the dock.
Is it possible to customize the apps shown in the launcher?
I'm currently forced to type the name of the app I want to start (if I can remember it) - my screen only shows 15 apps and the ones I need aren't in the short list.
Would also be cool to have multiple "folders" to choose from similar to what can be done in Android/iOS.

**Comment by jackleberryfin** [comment]
I think categories is not that helpful, however simply sorting by most frequently used might be.

**Reply by PapyElGringo** [reply]
@jackleberryfin it's should already be the case
